tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -197,15 +197,14 @@ def test_as_no_org_read_user(self):
             assert "orgRoleList" not in response.data
 
     def test_as_superuser(self):
-        self.user = self.create_user("super@example.org", is_superuser=True)
-        org = self.create_organization(owner=self.user)
-        team = self.create_team(name="appy", organization=org)
+        self.superuser = self.create_user(is_superuser=True)
+        team = self.create_team(name="appy", organization=self.organization)
 
-        self.login_as(user=self.user)
+        self.login_as(user=self.superuser, superuser=True)
```

Comment:
This test was not actually testing superuser but org owner b/c we weren't logging in as an active superuser